deployment-transformer/pom.xml

```
@@ -61,7 +61,7 @@
         <maven.compiler.source>${java.version}</maven.compiler.source>
         <maven.compiler.target>${java.version}</maven.compiler.target>
 
-        <payara.version>6.2022.1.Alpha4</payara.version>
+        <payara.version>6.2023.1</payara.version>
```

Since we have to publish this anyway, might as well make it the latest.
```suggestion
        <payara.version>6.2023.2</payara.version>
```